third_party/terraform/website/docs/d/google_kms_crypto_key.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "google"
+page_title: "Google: google_kms_crypto_key"
+sidebar_current: "docs-google-kms-crypto-key-x"
```

Comment:
Can you change the side_bar to `docs-google-datasource-kms-crypto-key`. Also feel free to add the 2 new docs files to website/google.erb . This is so that the links show up in the side bar, and when clicked they will highlight the current page in purple and the parent section based on a prefix match of the `sidebar_current` field (hence adding the -datasource- in there).
[example here](https://github.com/terraform-providers/terraform-provider-google/blob/master/website/google.erb#L51-L52)